tabpy/tabpy_server/app/app_parameters.py

```diff
@@ -10,7 +10,6 @@ class ConfigParameters:
     TABPY_TRANSFER_PROTOCOL = "TABPY_TRANSFER_PROTOCOL"
     TABPY_CERTIFICATE_FILE = "TABPY_CERTIFICATE_FILE"
     TABPY_KEY_FILE = "TABPY_KEY_FILE"
-    TABPY_PWD_FILE = "TABPY_PWD_FILE"
```

Comment:
intentional?

Reply:
Yes, this was a duplicate parameter. You can see its copy at the top of the list.